README.md

```diff
@@ -42,3 +42,18 @@ This is an application that I wrote long ago, but have since repurposed to play
 
 - Tests can be run using `rake test`
 - For test coverage, run your tests, open up `coverage/index.html` in your browser
+
+## Docker Deploy
+
+* [Install docker](https://docs.docker.com/engine/installation/)
+* [Install docker-compose](https://docs.docker.com/compose/install/)
+
+Run the following three commands in the project's root directory:
+
+```
+docker-compose build
+docker-compose run web rake db:setup
+docker-compose up
```

Comment:
The app works, but complains about filesystem things for log files and doesn't actually log anything. Do you know anything about that? This could be that my local filesystem is HFS+, whereas the image is using a different one. Just guessing here.
![screen shot 2017-01-25 at 15 27 35](https://cloud.githubusercontent.com/assets/2058614/22307930/efca3172-e312-11e6-8d08-cceda2241275.png)